dpgen/generator/arginfo.py

```diff
@@ -194,6 +195,8 @@ def model_devi_lmp_args() -> List[Argument]:
                  doc=doc_model_devi_f_avg_relative),
         Argument("model_devi_clean_traj", [
                  bool, int], optional=False, doc=doc_model_devi_clean_traj),
+        Argument("model_devi_merge_traj", [
+                 bool], optional=False, doc=doc_model_devi_merge_traj),
```

Comment:
`optional=True, default=False`

Reply:
I'm wondering that the argument of `model_devi_clean_traj` also need to be modified? `optional=True, default=True`

Reply:
https://github.com/deepmodeling/dpgen/blob/8dea29ef125f66be9641afe5ac4970433a9c9ce1/dpgen/generator/run.py#L3526

Reply:
Yes